src/Views/Component.elm

```diff
@@ -360,7 +379,7 @@ elementTransformsView config component index transformsResults transforms =
 
                     -- Note: allows truncated ellipsis in table cells https://stackoverflow.com/a/11877033/330911
                     , style "max-width" "0"
-                    , title <| Process.getDisplayName transform
+                    , title transform.name
```

Comment:
Oh nice! If I get it right, it allows you to display the `displayName` in the cell and at the same time having the process name on hover?

Reply:
yes! Best of both worlds :)